pkg/windows/windows.go

```diff
@@ -120,13 +61,73 @@ const (
 	ManagedTag = "OpenShift managed"
 	// containersFeatureName is the name of the Windows feature that is required to be enabled on the Windows instance.
 	containersFeatureName = "Containers"
-	// wicdCAFile is the name of the file that holds the WICD service account CA certificate
-	wicdCAFile = K8sDir + "sa-ca.crt"
-	// wicdTokenFile is the name of the file that holds the WICD service account secret token
-	wicdTokenFile = K8sDir + "sa-token"
 )
 
 var (
+	// GcpGetHostnameScriptRemotePath is the remote location of the PowerShell script that resolves the hostname
```

Comment:
If we are moving away from these being constants, we should look to see if all of the below need to be global variables.

Reply:
Some definitely don't, such as `WindowsExporterServiceCommand`. I was worried about including too much within this bug fix, let me know if you think thats within scope here.

Reply:
I would then suggest to fix them in place in the constants without introducing these global variables. i.e. just remove the trailing back slashes where needed and not change all the constants. Alternatively you can do this in the command by removing a suffix. You can then open a follow up PR to fix it up correctly.